zipline/data/data_portal.py

```diff
@@ -198,7 +198,7 @@ def __init__(self,
                 if reader is not None
             ]
             if last_minutes:
-                self._last_available_minute = min(last_minutes)
+                self._last_available_minute = max(last_minutes)
```

Comment:
I would have guessed that this was originally `min` because that would be the last time we have data for both `equity_minute_reader` and `future_minute_reader`. Do we want to run backtests after one reader ends, but while the other still has data?

Reply:
We already do! On prod right now, you cannot use `history` after the equities market closes on the current day. imo that should just fill NaN for equities or ffill the price field.